pytest_reportportal/plugin.py

```diff
@@ -121,6 +130,15 @@ def pytest_sessionfinish(session):
     if is_master(session.config):
         session.config.py_test_service.finish_launch(status='RP_Launch')
 
+    print(
```

Comment:
From my point of view, this message is excessive. I'd leave the only message at the begging of the session.

Reply:
There is a problem here, launch creates asyncronous, and it can lead to missing launch_id at startup.

how can I solve this?

Reply:
i guess it makes sense to have it at finish as well.
in most cases, engineers look at final numbers. Link available at the end of session can also be helpful.

Reply:
> There is a problem here, launch creates asyncronous, and it can lead to missing launch_id at startup.

What is the purpose of lines 70-78 in this case? I see the same launch id both in the beginning and at the end of the session on your [screenshot](https://user-images.githubusercontent.com/20853475/68474845-938a6980-0237-11ea-98f7-18b90816989c.png).